src/AttachmentPreview/AttachmentPreview.less

```diff
@@ -0,0 +1,195 @@
+@import (reference) "../less/utilities";
+
+@HEADER_BAR_HEIGHT: @size_5xl;
+@DESKTOP_BUFFER_SIZE: @size_l;
+@MOBILE_BUFFER_SIZE: @size_xs;
+
+// TODO: can I add this to sizing.less?
+@size_ml: 1.25rem;
+
+.AttachmentPreview--Background {
+  .zIndex--5;
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  background: fade(@neutral_black, 60%);
+}
+
+.AttachmentPreview--HeaderBar {
+  .zIndex--6;
+  background-color: @neutral_black;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: @HEADER_BAR_HEIGHT;
+  .alignItems(center);
+  color: @neutral_white;
+}
+
+.AttachmentPreview--AttachmentName {
+  line-height: @size_l;
+  // TODO: why isn't this adding the ellipsis?
```

Comment:
I think we resolved a similar issue elsewhere recently, but I can't quite remember what the details were!

Think I recall that @cofauver and @nbhatia823 were involved? 🤔

Reply:
I think I ended up needing to adjust a parent container (not the element with the misbehaving text, but it's parent) to change the `overflow` property? Unfortunately, this was one of so many changes that I don't remember the specifics.

Reply:
Also have a similar memory that the parent container needs to have a new property, but thinking it is that the `width: 100%` is what might do it.  So maybe try it in `AttachmentName` and/or `HeaderBar`. Hopefully either this or `overflow` as Cory mentioned will work!